### ISSUE TITLE
a11y EPUB display guide: missing legal section

### ISSUE THREAD

**Issue by panaC** [edrlab/thorium-reader#2940]
Follow up this [ISSUE](https://github.com/edrlab/thorium-reader/issues/2536) and [PR](https://github.com/edrlab/thorium-reader/pull/2937)

https://w3c.github.io/publ-a11y/a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html#legal-considerations

`a11y:exemption` not implemented in [r2-shared-js ](https://github.com/readium/r2-shared-js/blob/3dbce230c09c00042b38d5dbc9ffba6f2420992e/src/models/metadata-accessibility.ts)

**Comment by danielweck** [comment]
https://github.com/readium/webpub-manifest/blob/4c73f7323f9241e61bb919ecae2656a491ba15f6/schema/a11y.schema.json#L18

```json
    "exemption": {
      "format": "string",
      "enum": [
        "eaa-disproportionate-burden",
        "eaa-fundamental-alteration",
        "eaa-microenterprise"
      ]
    }
```
